### ISSUE TITLE
Apply header values to all raw events

### ISSUE THREAD

**Issue by davidpitkin** [carbonblack/cbapi#80]
It looks like not all raw messages have all the header messages like process_path and process_md5. 

Here's an example https://github.com/carbonblack/cbapi/blob/master/server_apis/proto/README.md the childproc os x messages do not have those values.

**Comment by jgarman** [comment]
Good catch - I will forward this on to the product team to evaluate. What version of the Cb Response server and what version of the Mac sensor are you running?

Unfortunately I don't have the ability to change the messages that are generated by the product but we need to make sure that any unexpected behavior is appropriately documented.
